File: src/main/java/com/graqr/threshr/model/redsky/products/Statistics.java
package com.graqr.threshr.model.redsky.products;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record Statistics(
        @JsonProperty("not_recommended_count") Long notRecommendedCount,
        @JsonProperty("question_count") Long questionCount, Rating rating,
        @JsonProperty("recommended_count") Long recommendedCount,
        @JsonProperty("recommended_percentage") Long recommendedPercentage,
        @JsonProperty("review_count") Long reviewCount) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/Statistics.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class Statistics(
    @param:JsonProperty("not_recommended_count") val notRecommendedCount: Long,
    @param:JsonProperty("question_count") val questionCount: Long,
    val rating: Rating,
    @param:JsonProperty("recommended_count") val recommendedCount: Long,
    @param:JsonProperty("recommended_percentage") val recommendedPercentage: Long,
    @param:JsonProperty("review_count") val reviewCount: Long
) {}
